themes/navy/layout/partial/head.njk

```diff
@@ -39,6 +39,8 @@
   <!-- Open Graph -->
   {{ open_graph({
     fb_admins: config.fb_admins,
+    image: url_for(page.og_image or 'icon/og-image-wide.png'),
```

Comment:
Also, no need `or` here. We just add `image: url_for('icon/og-image-wide.png')` :)

Reply:
fixed with https://github.com/hexojs/site/pull/2154/commits/89563602f4b312e2511c2934dfa17d25755ce151